tools/failures.py

```diff
@@ -113,19 +113,28 @@ def format_in_table(active_jobs, master):
     print "Total jobs %s" % (sum_removed + sum_remaining)
 
 
-def insert_in_database(to_remove, date=None):
+def insert_in_database(to_insert, date=None):
     if not date:
         date = datetime.datetime.now().strftime('%Y-%m-%d')
     else:
         date = date.strftime('%Y-%m-%d')
 
     run_query('delete from seta where date="%s"' % date)
-    for jobtype in to_remove:
+    for jobtype in to_insert:
         query = 'insert into seta (date, jobtype) values ("%s", ' % date
         query += '"%s")' % jobtype
         run_query(query)
 
 
+def prepare_the_database():
+    # wipe up the job data older than 90 days
+    date = datetime.datetime.now() - datetime.timedelta(days=90)
```

Comment:
I would name it: SETA_WINDOW=90

where window refers to a window in time :)

On Thu, Jun 9, 2016 at 8:33 AM, Tiramisu 1993 <notifications@github.com>
wrote:

> In tools/failures.py
> <https://github.com/dminor/ouija/pull/88#discussion_r66431801>:
>
> >          query = 'insert into seta (date, jobtype) values ("%s", ' % date
> >          query += '"%s")' % jobtype
> >          run_query(query)
> >
> >
> > +def prepare_the_database():
> > +    # wipe up the job data older than 90 days
> > +    date = datetime.datetime.now() - datetime.timedelta(days=90)
>
> Hmm, what should we named this constant? Just DAYS? (sorry, I'm really not
> good at naming
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dminor/ouija/pull/88/files/b7a357810085cd093ead4d52e5ab6ebd6eef99ef#r66431801>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAD0fKLcnDR7_gMQWkoRXEGiunuMMWt8ks5qKAgxgaJpZM4IxuIs>
> .
>